### DISCUSSION TITLE
Animated GIFs?

### DISCUSSION THREAD

**Discussion by NeilAgg2001** [LibrePDF/OpenPDF#851] [Q&A] [answered]
I was wondering if I can embed an animated gif into a PDF.

If I do, will the PDF show the first frame or the full animation?

**Comment by asturio** [comment]
Have you tried it? I would also be interested in it. Maybe it depends from the viewer?

**Reply by NeilAgg2001** [reply]
I tried it, the gif only showed the first frame.

**Comment by asturio** [comment, marked as answer]
Here is a nice discussion about it.

https://stackoverflow.com/questions/9009771/is-it-possible-to-embed-animated-gifs-in-pdfs

**Reply by NeilAgg2001** [reply]
I don't see a link to a discussion.

**Reply by asturio** [reply]
Just updated the comment to have a link.
